plugin/core/edit.py

```diff
@@ -55,41 +56,29 @@ def open_and_apply_edits(self, path, file_changes):
 
 class LspApplyDocumentEditCommand(sublime_plugin.TextCommand):
     def run(self, edit, changes: 'Optional[List[dict]]' = None, show_status=True):
+        # Apply the changes in reverse, so that we don't invalidate the range
+        # of any change that we haven't applied yet.
+        start = timer()
+        changes2 = changes or []
```

Comment:
you could also just call this `changes` again: `changes = changes or []`, no?

Reply:
Unfortunately not. Since `changes` might be null, mypy won't allow `changes[index]`. So I introduced another variable with a different type (not `Optional`).

Would appreciate a better solution.